docs/howto/deploy/aws.md

```diff
@@ -224,10 +229,65 @@ Checkout Nginx [documentation](https://kubernetes.github.io/ingress-nginx/user-g
         This way, lakeFS will be able to create repositories only under this specific path (see: [Storage Namespace][understand-repository]).
       * lakeFS will try to assume the role `[IAM_ROLE]`.
    </div>
+   <div markdown="1" id="bucket-policy-express">
+
+      In case you like to allow also S3 Express One Zone's directory bucket, the following include `lakeFSDirectoryBucket` segment to allow access to the bucket.
```

Comment:
```suggestion
      In case you like to allow also S3 Express One Zone's directory bucket, use the following policy. Note the `lakeFSDirectoryBucket` statement which is specifically required to use this feature.
```